docs/heasarc/heasarc.rst

```diff
@@ -11,292 +11,284 @@ This is a python interface for querying the
 `HEASARC <https://heasarc.gsfc.nasa.gov/>`__
 archive web service.
 
-The capabilities are currently very limited ... feature requests and contributions welcome!
+The main interface for the Heasarc services``heasarc.Heasarc`` now uses
+Virtual Observatory protocols with the Xamin interface, which offers
```

Comment:
please add a link for `Xamin`, it meant nothing to me and that maybe the case for other users, too.